board/post-image.sh

```diff
@@ -90,6 +90,9 @@ __EOF__
 			cat "$BR2_EXTERNAL_PICAM_PATH/camera.txt" >> "${BINARIES_DIR}/camera.txt"
 		fi
 
+		# Add default video_formats.txt
+                cp "$BR2_EXTERNAL_PICAM_PATH/package/piwebcam/video_formats.txt" "${BINARIES_DIR}/video_formats.txt"
```

Comment:
This should be part of `package/piwebcam/piwebcam.mk`. Any reason why you chose `video_formats` instead of `resolutions`. Happy with both, it's just a longer name.

Reply:
My idea of the fix was to be able to configure the piwebcam to ignore resolutions after the image has been burned to an SD card.

So if you see that the resolution is too low, you can put the SD card into your PC and change the `video_formats.txt` file on the `/boot` partition, reinsert it back into your raspberrypi and it will work.

The reason why I changed the name was that now not only the resolution gets configurable, but also the streaming format. So I thought naming the file just `resolutions.txt` is misleading.

Reply:
Right, this approach is sensible. What I meant however was to use the Makefile instead of the shell script to add that file.

Reply:
I have looked into this a bit more. But I did not find out how to place files in the `/boot` partition out of a buildroot package. I understood how to place files into the staging area to be included in the image, but I am not sure if this counts for the `/boot` partition, as well...

Reply:
Oh you don't have to, /etc would be the right place for this. A user should then be able to overwrite that file from /boot. Just add this line to `package/piwebcam/piwebcam.mk`

```
$(INSTALL) -D -m 644 $(PIWEBCAM_PKGDIR)/video_formats.txt $(TARGET_DIR)/etc/video_formats
```

Nevertheless I think we should come up with a better name for this file?

Reply:
I have implemented the installation in 618e516 and I am open to suggestions for another file name.